exercises/queen-attack/tests/queen-attack.rs

```diff
@@ -3,79 +3,84 @@ extern crate queen_attack;
 use queen_attack::*;
 
 #[test]
-fn test_queen_creation_with_valid_position() {
-    let white_queen = Queen::new((2,4));
-    assert!(white_queen.is_ok());
+fn chess_position_on_the_board_is_ok() {
+    assert!(ChessPosition::new(2, 4).is_ok());
 }
 
 #[test]
 #[ignore]
-fn test_queen_creation_with_incorrect_positions() {
-    let white_queen = Queen::new((-1,2));
-    assert!(white_queen.is_err());
-
-    let white_queen = Queen::new((8,2));
-    assert!(white_queen.is_err());
+fn chess_position_off_the_board_is_err() {
+    assert!(ChessPosition::new(-1, 2).is_err());
+    assert!(ChessPosition::new(8, 2).is_err());
+    assert!(ChessPosition::new(5, -1).is_err());
+    assert!(ChessPosition::new(5, 8).is_err());
+}
 
-    let white_queen = Queen::new((5,-1));
-    assert!(white_queen.is_err());
+#[test]
+#[ignore]
+fn queen_is_created_with_a_ok_position() {
+    Queen::new(ChessPosition::new(2, 4).unwrap());
+}
 
-    let white_queen = Queen::new((5,8));
-    assert!(white_queen.is_err());
+#[test]
+#[ignore]
+#[should_panic(expected = "Invalid Position")]
```

Comment:
I read https://doc.rust-lang.org/book/testing.html and it says "The test harness will make sure that the failure message **contains** the provided text." (emphasis mine) so that addresses what I was thinking about.